packages/block-library/src/gallery/save.js

```diff
@@ -46,6 +46,22 @@ export default function save( { attributes } ) {
 					</li>
 				);
 			} ) }
+		</>
+	);
+
+	if ( galleryCaption ) {
+		return (
```

Comment:
Could we always wrap in a `figure` tag, for consistency? We do the same for all other media (image, audio, video, embed...)